Added clarification on validation technical profile execution v2 (#96058)
Azure/azure-content: articles/active-directory-b2c/self-asserted-technical-profile.md

You can also call a REST API technical profile with your business logic, overwrite input claims, or enrich user data by further integrating with corporate line-of-business application. For more information, see [Validation technical profile](validation-technical-profile.md)

> [!NOTE]

> > A validation technical profile will only be triggered when there is some input from the user. It is not possible to create an _empty_ self-asserted technical profile that exists to call validation technical profiles only e.g. to take advantage of the **ContinueOnError** attribute of a **ValidationTechnicalProfile** element. These validation technical profiles must be called from a self-asserted technical profile that solicits some input from the user, or called from an orchestration step in a user journey.

Reviewer: ```suggestion
> A validation technical profile is only triggered when there's an input from the user. You can't create an _empty_ self-asserted technical profile to call a validation technical profile just to take advantage of the **ContinueOnError** attribute of a **ValidationTechnicalProfile** element. You can only call a validation technical profile from a self-asserted technical profile that requests an input from the user, or from an orchestration step in a user journey. 
```

Reviewer: @stuart938503 

Please see this suggested change. If it communicates your original meaning, please commit it, and I'll sign of your PR.
